docs/crio.conf.5.md

```diff
@@ -337,6 +337,32 @@ A workload is chosen for a pod based on whether the workload's **activation_anno
   "io.kubernetes.cri-o.ShmSize" for configuring the size of /dev/shm.
   "io.kubernetes.cri-o.UnifiedCgroup.$CTR_NAME" for configuring the cgroup v2 unified block for a container.
   "io.containers.trace-syscall" for tracing syscalls via the OCI seccomp BPF hook.
+  "io.kubernetes.cri-o.seccompNotifier" for enabling the seccomp notifier feature.
+
+#### Using the seccomp notifier feature:
+
+This feature can help you to debug seccomp related issues, for example if
+blocked syscalls (permission denied errors) have negative impact on the
+workload.
+
+To be able to use this feature, configure a runtime which has the annotation
+"io.kubernetes.cri-o.seccompNotifier" in the `allowed_annotations` array.
```

Comment:
sorry should have mentioned it earlier: what do you think about having this annotation be `seccompNotifierAction` and have it take an action. we can start with `stop` or `ignore/""` but maybe in the future we'd want another action

Reply:
Alright, changed as suggested :+1: